src/mumble/API_v_1_x_x.cpp

```diff
@@ -1609,9 +1609,14 @@ void MumbleAPI::log_v_1_0_x(mumble_plugin_id_t callerID, const char *message,
 
 void MumbleAPI::playSample_v_1_0_x(mumble_plugin_id_t callerID, const char *samplePath,
 								   std::shared_ptr< api_promise_t > promise) {
+	playSample_v_1_2_x(callerID, samplePath, 1.0f, promise);
+}
+
+void MumbleAPI::playSample_v_1_2_x(mumble_plugin_id_t callerID, const char *samplePath, float volume,
+								   std::shared_ptr< api_promise_t > promise) {
 	if (QThread::currentThread() != thread()) {
 		// Invoke in main thread
-		QMetaObject::invokeMethod(this, "playSample_v_1_0_x", Qt::QueuedConnection, Q_ARG(mumble_plugin_id_t, callerID),
+		QMetaObject::invokeMethod(this, "playSample_v_1_2_x", Qt::QueuedConnection, Q_ARG(mumble_plugin_id_t, callerID),
 								  Q_ARG(const char *, samplePath), Q_ARG(std::shared_ptr< api_promise_t >, promise));
```

Comment:
This seems to be missing the new volume parameter

Reply:
Yes. I fixed that in the latest commit.